src/libxtp/gwbse/bse_operator.cc

```diff
@@ -94,11 +94,11 @@ Eigen::RowVectorXd BSE_OPERATOR<cqp, cx, cd, cd2>::Hqp_row(Index index) const {
   Index v1 = vc.v(index);
   Index c1 = vc.c(index);
   Eigen::MatrixXd Result = Eigen::MatrixXd::Zero(_bse_ctotal, _bse_vtotal);
-  Index cmin = _bse_cmin - _opt.qpmin;
+  Index cmin = _bse_vtotal;
   // v->c
   Result.col(v1) = _Hqp.col(c1 + cmin).segment(cmin, _bse_ctotal);
   // c-> v
-  Index vmin = _opt.vmin - _opt.qpmin;
+  Index vmin = 0;  //_opt.vmin - _opt.qpmin;
```

Comment:
why zero?

Reply:
The HQP matrix that is used to initialize the BSE object now should exactly contain only the entries from vmin:cmax.

Reply:
Can you add the comment then please.

Reply:
Where?